libexec/erlang

```diff
@@ -76,9 +76,22 @@ require_deploy_config() {
     REQUIRED_CONFIGS+=("TEST_AT")
   fi
 
+  __detect_release_store_type
+  if [ "$RELEASE_STORE_TYPE" = "s3" ]; then
+    REQUIRED_CONFIGS+=("AWS_ACCESS_KEY_ID")
+    REQUIRED_CONFIGS+=("AWS_SECRET_ACCESS_KEY")
+  fi
+
   OPTIONAL_CONFIGS+=("VERSION")
 }
 
+s3_requirements() {
+  __detect_release_store_type
+  if [ "$RELEASE_STORE_TYPE" = "s3" ]; then
+    __find_remote_s3cmd
```

Comment:
Are there any chances that this procedure will be used anywhere else? It doesn't look too big to inline to me.

Reply:
I created `s3_requirements` in order not to copy paste the same line in strategies over and over again. If you think that it shouldn't be a procedure I will inline it.